sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -137,33 +137,78 @@ Recognize data from US sales receipts using a prebuilt model.
 using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
 {
     RecognizedReceiptCollection receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
+
     foreach (var receipt in receipts)
     {
-        USReceipt usReceipt = receipt.AsUSReceipt();
+        FormField merchantNameField;
+        if (receipt.RecognizedForm.Fields.TryGetValue("MerchantName", out merchantNameField))
+        {
+            if (merchantNameField.Value.Type == FieldValueType.String)
+            {
+                string merchantName = merchantNameField.Value.AsString();
 
-        string merchantName = usReceipt.MerchantName?.Value ?? default;
-        DateTime transactionDate = usReceipt.TransactionDate?.Value ?? default;
-        IReadOnlyList<USReceiptItem> items = usReceipt.Items ?? default;
-        float subtotal = usReceipt.Subtotal?.Value ?? default;
-        float tax = usReceipt.Tax?.Value ?? default;
-        float tip = usReceipt.Tip?.Value ?? default;
-        float total = usReceipt.Total?.Value ?? default;
+                Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {merchantNameField.Confidence}");
+            }
+        }
 
-        Console.WriteLine($"Recognized USReceipt fields:");
-        Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {usReceipt.MerchantName.Confidence}");
-        Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {usReceipt.TransactionDate.Confidence}");
+        FormField transactionDateField;
+        if (receipt.RecognizedForm.Fields.TryGetValue("TransactionDate", out transactionDateField))
+        {
+            if (transactionDateField.Value.Type == FieldValueType.Date)
+            {
+                DateTime transactionDate = transactionDateField.Value.AsDate();
 
-        for (int i = 0; i < items.Count; i++)
+                Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {transactionDateField.Confidence}");
+            }
+        }
+
+        FormField itemsField;
+        if (receipt.RecognizedForm.Fields.TryGetValue("Items", out itemsField))
         {
-            USReceiptItem item = usReceipt.Items[i];
-            Console.WriteLine($"    Item {i}:  Name: '{item.Name.Value}', Quantity: '{item.Quantity?.Value}', Price: '{item.Price?.Value}'");
-            Console.WriteLine($"    TotalPrice: '{item.TotalPrice.Value}'");
+            if (itemsField.Value.Type == FieldValueType.List)
+            {
+                foreach (FormField itemField in itemsField.Value.AsList())
+                {
+                    if (itemField.Value.Type == FieldValueType.Dictionary)
+                    {
+                        IReadOnlyDictionary<string, FormField> itemFields = itemField.Value.AsDictionary();
+
+                        FormField itemNameField;
+                        if (itemFields.TryGetValue("MerchantName", out itemNameField))
```

Comment:
I believe for items we expect to see field names from `Name` , `Quantity`, `Price` or `TotalPrice`.

Reply:
Thanks for pointing this out. This was a mix of copy-paste mistakes + carelessness on my part.